xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -1160,6 +1160,14 @@ INFO_RET CMusicInfoScanner::DownloadAlbumInfo(const CAlbum& album, const ADDON::
       }
       Sleep(1);
     }
+    /*
+    Finding album using xml scraper may request data from Musicbrainz.
+    MusicBrainz rate-limits queries to 1 per sec, once we hit the rate-limiter the server
+    returns 503 errors for all calls from that IP address.
+    To stay below the rate-limit threshold wait 1s before proceeding
+    */
+    if (!info->IsPython())
```

Comment:
Sorry no, m_isPython is the only flag CScraper has, and IsPython the only method of this kind. 

If we do add support of other kinds of scraper a lot more would need to change anyway, a refactoring of scraping would be sensible